cluster/points_writer.go

```diff
@@ -274,32 +278,32 @@ func (w *PointsWriter) writeToShard(shard *meta.ShardInfo, database, retentionPo
 	var wrote int
 	timeout := time.After(w.WriteTimeout)
 	var writeError error
-	for _, nodeID := range shard.OwnerIDs {
+	for i := 0;  i < len(shard.OwnerIDs); i++ {
```

Comment:
Why did this change?  I don't see us using `i`, but I could be missing it.  The original is much clearer and idiomatic in Go.

If it was to stop using the `nodeID`, you can simply do:
```go
for range shard.OwnerIDS {
```